common/src/main/java/org/jboss/hal/testsuite/fragment/WindowFragment.java

```diff
@@ -110,4 +117,13 @@ public String getTitle() {
         return title.getText();
     }
 
+    protected void clickButtonAndLogIfFails(String label) {
+        try {
+            clickButton(label);
+        } catch (WebDriverException e) {
+            log.debug("Button with label \"" + label + "\" not found");
+            throw e;
+        }
```

Comment:
from method name I would guess that in case of fail it will only log in case of failure. But exception is thrown as well => I suggest to use more clear name of the method.